content/blog/2022/09/2022-09-18-digital-ocean-sponsorship.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Digital Ocean and Jenkins partnership continues to grow!"
+tags:
+- jenkins
+- digitalocean
+- community
+author: kmartens27
+description: "DigitalOcean increases Jenkins sponsorship"
+opengraph:
+  image: /images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png
+---
+
+We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
+Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.
```

Comment:
I wasn't sure if we should make it more personal by using "We"? 
```suggestion
We want to thank DigitalOcean for providing such incredible support to the Jenkins project.
```

Reply:
I definitely like that better @kellie-freeman, much smoother and natural to read.